rules/windows/process_creation/proc_creation_win_file_permission_modifications.yml

```diff
@@ -16,10 +17,17 @@ detection:
             - '\takeown.exe'
             - '\cacls.exe'
             - '\icacls.exe'
+            - '\net.exe' # Option available when used with "net share"
+            - '\net1.exe'
         CommandLine|contains: '/grant'
     selection2:
         Image|endswith: '\attrib.exe'
         CommandLine|contains: '-r'
+    selection3:
+        Image|endswith:
+            - '\net.exe'
+            - '\net1.exe'
+        CommandLine|contains:
```

Comment:
Empty?

Reply:
Nice catch

Reply:
Good catch. I was going to do it in it's own selection but then saw that "grant" is already there. I'll remove it